internal/model/metrics.go

```diff
@@ -0,0 +1,180 @@
+// Copyright (c) F5, Inc.
+//
+// This source code is licensed under the Apache License, Version 2.0 license found in the
+// LICENSE file in the root directory of this source tree.
```

Comment:
would like to see unit tests for some of the functions in this file, even if you can't test everything easily you can test some of it

Reply:
Adding `metrics_test.go`, which has 100% coverage

<img width="771" alt="image" src="https://github.com/nginx/agent/assets/85518764/4b51e003-66e7-4e01-8e1a-37815835bef4">